cloudify/plugin_installer.py

```diff
@@ -562,12 +562,19 @@ def get_pth_dir(venv=None):
 
 def _python_executable(plugin):
     if 'supported_py_versions' not in plugin:
+        # This is the case for source plugins
         return sys.executable
-    if 'py310' in plugin['supported_py_versions']:
+
+    v = sys.version_info
+    if f'py{v.major}{v.minor}' in plugin['supported_py_versions']:
         return sys.executable
-    if 'py36' in plugin['supported_py_versions']:
-        return '/usr/bin/python3.6'
+    for v in reversed(plugin['supported_py_versions']):
```

Comment:
What does the content of plugin['supported_py_versions'] look like? Is it a string?

Reply:
it's a list of `["py36", "py310"]`. Yes, without the dot. It is stupid, yes.

Reply:
cf https://github.com/cloudify-cosmo/wagon/blob/4019fbc28e915da5b522abcdaefece2493409a4d/wagon.py#L417

Reply:
no, it's a list like `['py36']`

Reply:
OK, makes sense then- lgtm.